filebeat/docs/fields.asciidoc

```diff
@@ -1435,6 +1436,339 @@ example: doc
 
 Type
 
+--
+
+[[exported-fields-haproxy]]
+== haproxy fields
+
+haproxy Module
+
+
+
+[float]
+== haproxy fields
+
+
+
+
+[float]
+== http fields
+
+Please add description
+
+
+*`haproxy.http.process_name`*::
++
+--
+Name of the process
+
+--
+
+*`haproxy.http.pid`*::
++
+--
+type: long
+
+Process ID
+
+--
+
+*`haproxy.http.client_ip`*::
++
+--
+client_ip is the IP address of the client which initiated the TCP connection to haproxy.
+
+--
+
+*`haproxy.http.client_port`*::
++
+--
+type: long
+
+client_port is the TCP port of the client which initiated the connection.
+
+--
+
+*`haproxy.http.frontend_name`*::
++
+--
+frontend_name is the name of the frontend (or listener) which received and processed the connection.
+
+--
+
+*`haproxy.http.backend_name`*::
++
+--
+backend_name is the name of the backend (or listener) which was selected to manage the connection to the server.
+
+--
+
+*`haproxy.http.server_name`*::
++
+--
+server_name is the name of the last server to which the connection was sent.
+
+--
+
+*`haproxy.http.time_client_req`*::
++
+--
+type: long
+
+time_client_req is the total time in milliseconds spent waiting for a full HTTP request from the client (not counting body) after the first byte was received.
+
+--
+
+*`haproxy.http.time_queue`*::
++
+--
+type: long
+
+time_queue is the total time in milliseconds spent waiting in the various queues.
+
+--
+
+*`haproxy.http.time_backend_connect`*::
++
+--
+type: long
+
+time_backend_connect is the total time in milliseconds spent waiting for the connection to establish to the final server, including retries.
+
+--
+
+*`haproxy.http.time_server_response`*::
++
+--
+type: long
+
+time_server_response is the total time in milliseconds spent waiting for the server to send a full HTTP response, not counting data.
+
+--
+
+*`haproxy.http.time_duration`*::
++
+--
+type: long
+
+time_duration is the time the request remained active in haproxy, which is the total time in milliseconds elapsed between the first byte of the request was received and the last byte of response was sent.
+
+--
+
+*`haproxy.http.status_code`*::
```

Comment:
Could you follow `ECS` in fields naming if possible? for example [http response fields](https://github.com/elastic/ecs#http) could be at the root level under `http.response...`.

Reply:
Will do.